crawler-user-agents.json

```diff
@@ -186,8 +186,8 @@
     "pattern": "mlbot"
   },
   {
-    "pattern": "yandexbot",
```

Comment:
in this pattern not used anymore? may we should ave two entries with two different patterns.